src/lib/markers/implementations/CursorMarker.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { MarkerCanvasConsumer } from '../MarkerCanvasContext'
+
+const criticalStyles = {
+  position: 'absolute',
+  top: 0,
+  bottom: 0,
+  // FIXME: are these critical styles?
+  width: '2px',
+  backgroundColor: 'pink'
```

Comment:
if these are not critical maybe they should be passed as props and they would have default values as specified here?

Reply:
`styles` are passed to the custom renderer so if the user wants to override width and color they can do so,  I'm still not 100% on this one though.